webserver/src/helpers/helpers.js

```
@@ -11,6 +12,8 @@ const Reply = db.reply;
 const Tag = db.tag;
 const Activity = db.activity;
 const BlockingNotifications = db.blockingNotifications;
+const fs = require('fs/promises');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)